doc/sphinx-guides/source/developers/coding-style.rst

```diff
@@ -2,26 +2,38 @@
 Coding Style
 ============
 
-Like all development teams, the `Dataverse developers at IQSS <https://dataverse.org/about>`_ have their habits and styles when it comes to writing code. Let's attempt to get on the same page. :)
-
-.. contents:: |toctitle|
-	:local:
-
 Java
 ----
 
 Formatting Code
 ~~~~~~~~~~~~~~~
 
-Tabs vs. Spaces
-^^^^^^^^^^^^^^^
+NetBeans
+^^^^^^^^
+
+We highly recommend using Netbeans as a text editor. This makes it significantly easier to commit code that is well formatted and type safe. By default, NetBeans will warn you about some things, but if you want better warnings and code formatting, download this##################TODO ADD FILE######################### zip file, and import it to your nNtBeans settings by going to options, import, and selecting the file. These settings will cause NetBeans to automatically format code that you've worked on whenever you save, point out many common problems in code, and make sure that all indentation uses spaces.
+
+Alternatively, you can set the warnings yourself. To do so, go to tools-> options-> editor -> hints, and turn on the following
+    - Braces
+    - Error Fixes
+    - JDK Migration Support
+    - Probable Bugs
+    - Standard Javac Warnings
```

Comment:
I don't use all of these. Let's have a conversation about which to turn on and why.